cmd/migrate_intakes/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+	"gopkg.in/launchdarkly/go-server-sdk.v5/ldcomponents"
+	"gopkg.in/launchdarkly/go-server-sdk.v5/testhelpers/ldtestdata"
+
+	"github.com/cmsgov/easi-app/pkg/appconfig"
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/storage"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func noErr(err error) {
+	if err != nil {
+		fmt.Println("Error!")
+		fmt.Println(err)
+		panic("Aborting")
+	}
+}
+
+func makeStore(ldClient *ld.LDClient) *storage.Store {
+	config := testhelpers.NewConfig()
+
+	dbConfig := storage.DBConfig{
+		Host:           config.GetString(appconfig.DBHostConfigKey),
+		Port:           config.GetString(appconfig.DBPortConfigKey),
+		Database:       config.GetString(appconfig.DBNameConfigKey),
+		Username:       config.GetString(appconfig.DBUsernameConfigKey),
+		Password:       config.GetString(appconfig.DBPasswordConfigKey),
+		SSLMode:        config.GetString(appconfig.DBSSLModeConfigKey),
+		MaxConnections: config.GetInt(appconfig.DBMaxConnections),
+	}
+
+	store, storeErr := storage.NewStore(dbConfig, ldClient)
+	if storeErr != nil {
+		panic(storeErr)
+	}
+	return store
+}
+
+func migrateIntakes() {
+	zapLogger, err := zap.NewDevelopment()
+	noErr(err)
+
+	ctx := appcontext.WithLogger(context.Background(), zapLogger)
+
+	td := ldtestdata.DataSource()
+	td.Update(td.Flag("emit-to-cedar").BooleanFlag().VariationForAll(true))
```

Comment:
So this doesn't just migrate but also flips a LD flag for CEDAR? Or is that part of migrating where you're sending data to CEDAR?

Reply:
It doesn't flip the flag, but creates an LD client that has that flag set to `true` (basically ignoring whatever's in LD)